meteor-client/runtime-config.js

```diff
@@ -0,0 +1,4 @@
+__meteor_runtime_config__ = {
```

Comment:
From previous experience, people tend to get difficulties with what you've just said, because loading it as an initial configuration using `webpack`, as basic as it sound, is not an easy task. What people ended up doing was creating a `<script>` tag in the `index.html` file which will be loaded before everything else. I want to prevent this problem.